### DISCUSSION TITLE
Option to use original filename

### DISCUSSION THREAD

**Discussion by TheOneAndOnlyGarcia** [directus/directus#3279] [Archive]
Hello, 

I am currently switching from strapi to directus and I must say I am thrilled! However, file handling is very important for our website. From the documentation of the v8 I think it was possible to use the original file name. Is this provided in the current version or (planned version)?

Question: **How can I set that I use the original file name as storage file or at least as query id?**

thanks a lot!

**Comment by shi-rudo** [comment]
an option to manually set the id of the file during upload would be welcome
